### PR TITLE
SIAE Radio: add additional sensors and data

### PR DESCRIPTION
**Pull request by murrant**
Improved labels
Modulation
Rate
Serial Number
Hardware name

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by jozefrebjak** [comment]
@murrant Are you using these radios ? If yes which model. We have bad experiences with SNMP turnet on on some models. Thank you.

**Comment by murrant** [comment]
@jozefrebjak ALFOplus2, no problems with snmp so far

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-64-release-changelog-may-2020/12290/1

File: LibreNMS/OS/SmOs.php [hunk 1]
```diff
@@ -3,20 +3,97 @@
...
+                }
+            }
+
+            if (!empty($totalOids['rx'])) {
+                $sensors[] = new WirelessSensor(
+                    'rate',
+                    $this->getDeviceId(),
+                    $totalOids['rx'],
+                    'total-rx',
+                    $index,
```
Comment (Jellyfrog) [inline LibreNMS/OS/SmOs.php:69]: @murrant Sorry, hunting old potentiation issues here, `$index` comes from previous foreach loop, is this really correct?

File: LibreNMS/OS/SmOs.php [hunk 2]
```diff
@@ -3,20 +3,97 @@
...
+                );
+            }
+
+            if (!empty($totalOids['tx'])) {
+                $sensors[] = new WirelessSensor(
+                    'rate',
+                    $this->getDeviceId(),
+                    $totalOids['tx'],
+                    'total-tx',
+                    $index,
```
Comment (Jellyfrog) [inline LibreNMS/OS/SmOs.php:82]: @murrant Same here

File: LibreNMS/OS/SmOs.php (hunk 1 above)
Comment (murrant) [reply LibreNMS/OS/SmOs.php:69]: does not seem correct.

**Pull request merged**